setup/tue.bash_functions

```diff
@@ -719,7 +729,7 @@ function tue-robocup-update
     cd ~/.tue
     git pull --ff-only
 
-    tue-robocup-install-package picaso_4d_systems
+    tue-robocup-install-package picaso_4d_systems #WHY????
```

Comment:
Yes, can be removed .. We are not using this anymore and there is no use for that line being there